transform-plugins/docs/XMLParser-transform.md

```diff
@@ -0,0 +1,82 @@
+# XML Parser Transform
+
+Description
+-----------
+The XML Parser uses XPath to extract field from a complex XML Event. This should generally be used in conjunction with
+the XML Reader Batch Source. The XML Source Reader will provide individual events to the XML Parser and the XML Parser
+will be responsible for extracting fields from the events and mapping them to output schema.
+
+
+Use Case
+--------
+The transform takes input record that contain xml events/records, parses it using the xpath specified and returns a
+structured record according to the schema specified by user.
+For example, this plugin can be used in conjuction with XML Reader Batch Source to extract values from XMLNews documents
+and to create structured records which will be easier to query.
```

Comment:
The transform takes an input record that contain XML events/records, parses it using the XPath specified and returns a
structured record according to the specified schema. For example, this plugin can be used in conjuction with the XML
Reader Batch Source to extract values from XMLNews documents and create structured records which are easier to query.